hs/src/Reach/Connector/ETH_Solidity.hs

```diff
@@ -471,7 +472,7 @@ solLargeArg' dv la =
       where
         go i a = one (".elem" <> pretty i) <$> solArg a
     DLLA_Obj m ->
-      solLargeArg' dv $ DLLA_Struct $ M.toAscList m
+      solLargeArg' dv $ DLLA_Struct $ map (first ("_" <>)) $ M.toAscList m
```

Comment:
Can you abstract these three uses of `("_" <>)` into a function in case we need to change it?